setup.py

```diff
@@ -20,7 +17,7 @@
     url="https://github.com/xbmc/addon-check",
     download_url="https://github.com/xbmc/addon-check/archive/master.zip",
     packages=setuptools.find_packages(exclude=['script.test', 'tests*']),
-    install_requires=requirements,
+    install_requires=['pillow', 'requests', 'radon'],
```

Comment:
Why hardcore the dependencies here instead of keeping the `requirements.txt` file?

Reply:
I just thought that if we are using pipenv then we can get rid of the requirements.txt file([can be problematic](https://www.kennethreitz.org/essays/a-better-pip-workflow)). Plus it actually doesn't effect anything in wrong way. Does it ?

Reply:
I don't like that dependencies have now to be changed in two places.
Moving one of the places somewhere into a large file makes it even worse.
Can't you read the required dependencies from Pipfile?